k8s-subst

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script generates K8s yaml files from a set of templates.  To run it you need to 
+# define env vars IMAGE_PATH and IMAGE_TAG such that clearwater images can be found at
```

Comment:
Why have we gone with having to define these as env vars, rather than passing them in as arguments to the script?
I'm not sure which way i prefer, but having them as env vars feels like it  might carry some  risk of contamination? Or possibly of people assuming they can then later just change the env vars, without needing to remodify the yaml files